src/TraitsV2/TaDeepAliasMethod.class.st

```diff
@@ -0,0 +1,52 @@
+"
+I extend my superclass not only aliasing the method but also rewriting all the users of the method in my methods.
+I am useful when you want to rename a method. You alias deeply with me and then remove it.
+
+I implement the #@=  operator.
+"
+Class {
+	#name : #TaDeepAliasMethod,
+	#superclass : #TaAliasMethod,
+	#category : #'TraitsV2-Compositions'
+}
+
+{ #category : #testing }
+TaDeepAliasMethod >> changesSourceCode: aSelector [ 
+	
+	^ true
+]
+
+{ #category : #rewriting }
+TaDeepAliasMethod >> rewriteText: aSelector [
+	| stream |
+	aSelector numArgs == 0
```

Comment:
can be moved after the point where a write stream is created and 'self ' string added